internal/vault/operator_client.go

```diff
@@ -100,13 +99,15 @@ type purgeUnmanagedConfig struct {
 	Enabled bool `json:"enabled,omitempty"`
 	Exclude struct {
 		Policies bool `json:"policies,omitempty"`
+		Auths    bool `json:"auth,omitempty"`
```

Comment:
What do you think about using `AuthMethods` for explicitly since the purge is a critical config :thinking: 
I'm split between using the same names in the config (e.g. `auth` for authentication methods and `secrets` for Secrets Engines) or using more clear names like `AuthMethods` and `SecretsEngines` ... WDYT? :thinking:

Reply:
I am in favour of `Auths` since it reflects exactly what will be under the `externalConfig`. As you know under externalConfig currently it's `auth` not `authMethod`. Same applies for `secrets`.